static/app/views/insights/database/components/noDataMessage.tsx

```diff
@@ -56,7 +56,7 @@ export function NoDataMessage({Wrapper = DivWrapper, isDataAvailable}: Props) {
     <Wrapper>
       {!isDataAvailable &&
         tct(
-          'No queries found. Try updating your filters, or learn more about performance monitoring for queries in our [documentation:documentation].',
+          'No queries found. Try updating your filters, or learn more about tracing for queries in our [documentation:documentation].',
```

Comment:
```suggestion
          'No queries found. Try updating your filters, or learn more about tracing for queries in our [documentation:documentation].',
```
I think we can leave this as 'perf monitoring' here since it's talking about the general concept.